tidb-cloud/integrate-tidbcloud-with-zapier.md

```diff
@@ -0,0 +1,232 @@
+
+   ```sql
+   mysql> SELECT * FROM test.github_global_event;
+   +-------------+-------------+------------+-----------------+----------------------------------------------+--------+---------------------+
+   | id          | type        | actor      | repo_name       | repo_url                                     | public | created_at          |
+   +-------------+-------------+------------+-----------------+----------------------------------------------+--------+---------------------+
+   | 25324462424 | CreateEvent | shiyuhang0 | shiyuhang0/docs | https://api.github.com/repos/shiyuhang0/docs | True   | 2022-11-18 08:03:14 |
+   +-------------+-------------+------------+-----------------+----------------------------------------------+--------+---------------------+
+   1 row in set (0.17 sec)
+   ```
+
+### Step 5: Publish your zap
+
+Click **Publish** to publish your zap. You can see the zap is running in the [home page](https://zapier.com/app/zaps).
+
+![Publish the zap](/media/tidb-cloud/zapier/zapier-tidbcloud-publish.png)
+
+Now, this zap will automatically record all the global events from your GitHub account into TiDB Cloud.
+
+## Triggers & Actions
+
+[Triggers and actions](https://zapier.com/how-it-works) are the key concepts in Zapier. By combining different triggers and actions, you can create various automation workflows.
+
+This section introduces the triggers and actions provided by TiDB Cloud App on Zapier.
+
+### Triggers
+
+- New Cluster: triggers when a new cluster is created.
+- New Table: triggers when a new table is created.
+- New Row: triggers when new rows are created. Only fetches the recent 10000 new rows.
+- New Row (Custom Query): triggers when new rows are returned from a custom query that you provide.
+
+### Actions
+
+The following table lists the actions supported by TiDB Cloud App. Note that some actions need extra resources and you need to prepare the corresponding resources before using the action.
+
+| Action | Description | Resource |
+|---|---|---|
+| Find Cluster | Finds an existing Serverless tier or Dedicated tier. | None |
+| Create Cluster | Creates a new cluster. Only supports creating a free Serverless Tier cluster. | None |
+| Find Database | Finds an existing database. | A Serverless Tier cluster |
+| Create Database | Creates a new database. | A Serverless Tier cluster |
+| Find Table | Finds an existing Table. | A Serverless Tier cluster and a database |
+| Create Table | Creates a new table. | A Serverless Tier cluster and a database |
+| Create Row | Creates a new row. | A Serverless Tier cluster, a database, and a table |
+| Update Row | Updates an existing row. | A Serverless Tier cluster, a database, and a table |
+| Find Row | Finds a row in a table via a lookup column. | A Serverless Tier cluster, a database, and a table |
+| Find Row (Custom Query) | Finds a row in a table via a custom query the you provide. | A Serverless Tier cluster, a database, and a table |
+
```

Comment:
@shiyuhang0 I made a small change to the table. PTAL.

Reply:
get